cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -1,3 +1,4 @@
+/* global _ */
```

Comment:
Added because `eslint` tests were not passing without it.
Is this not required / to be handled differently?